README.md

```diff
@@ -8,17 +8,33 @@ Terraform Module for Creating S3 backed Websites
 
 ## Usage
 
+#### Create s3 website bucket
 ```terraform
 module "website" {
   source      = "git::https://github.com/cloudposse/tf_s3_website.git?ref=master"
   namespace   = "${var.namespace}"
   stage       = "${var.stage}"
   name        = "${var.name}"
   hostname    = "${var.hostname}"
-  dns_zone_id = "${var.dns_zone_id}"
 }
 ```
 
+#### Create s3 website bucket with associated CNAME of your (sub)domain hosted in Route53
```

Comment:
It's actually not a `CNAME` but `ALIAS` of type `A`

https://github.com/cloudposse/tf_vanity/blob/master/main.tf#L26

Reply:
Let's just write 
````
#### Create S3 Website Bucket with Route53 DNS
````